modules/c++/math/tests/complexBenchmark.cpp

```diff
@@ -0,0 +1,159 @@
+/* Users guide
+    
+    To use this test, you must run the executable with 3 extra numbers
+    the first number is the size of the initial array
+    the second number is the growth factor, each time the size is increased, the old size is multiplied by this number
+    the third number is the number of times the size should grow
+
+    example:  ./complexBenchmark 10 2 15
+    this will run the test for array sizes 10, 20, 40,... until the array size has been increased 15 times
+
+*/
+
+#include <cstdlib>
+#include <ctime>
+#include <iostream>
+#include <fstream>
+#include <iomanip>
+#include <sstream>
+#include <complex>
+#include <sys/StopWatch.h>
+
+std::complex<float> getMeanWComplex(sys::RealTimeStopWatch& wtch, std::complex<float>* in, long long sze, double& duration)
```

Comment:
Always use size_t rather than long long for this kind of thing... size_t will always be big enough to hold an array in memory